opentelemetry-api/src/opentelemetry/metrics/examples/pre_aggregated.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from opentelemetry.metrics import Meter
+from opentelemetry.metrics.label_key import LabelKey
+
+METER = Meter()
+LABEL_KEYS = [LabelKey("environment", 
+                       "the environment the application is running in")]
+SUM_METRIC = METER.create_int_counter("sum numbers", 
+                                      "sum numbers over time",
+                                      "number",
+                                      LABEL_KEYS)
+LABEL_VALUES = ["Testing"]
+SUM_TIME_SERIES = SUM_METRIC.getOrCreateTimeSeries(LABEL_VALUES)
```

Comment:
get_or_create_timeseries? I see a few examples that are using camelCase although the signature itself uses underscores.

Reply:
Good catch. I'm not compiling the example files as the API doesn't actually return anything as of yet.